### DISCUSSION TITLE
Move bug report item in load/save menu

### DISCUSSION THREAD

**Discussion by AaronVanGeffen** [OpenRCT2/OpenRCT2#21979] [Feature requests]
With #21855 introducing a separator between the 'Load' and 'Save' options, I've been wondering if we could improve the balance of this menu further. I think a fairly simple change can bring back some order.

Currently, the menu looks like this:
<img width="197" alt="Screenshot 2024-05-06 at 23 48 02" src="https://github.com/OpenRCT2/OpenRCT2/assets/604665/96ca7993-a063-4c81-aa69-bce9c2f698a2">

I'd like to suggest we make it look like this:
<img width="197" alt="Screenshot 2024-05-06 at 23 48 02 copy" src="https://github.com/OpenRCT2/OpenRCT2/assets/604665/5a63be1a-724e-445c-b10c-511b3b99cf73">

This groups the 'Bug report' item with 'About' and 'Options', and adds a separator between it and both of the screenshot options. The number of separators remains the same, but the items would be more balanced this way, I think.

Alternatively, we could move the screenshot options before the 'About' options as well.

Thoughts?

**Comment by Basssiiie** [comment]
I think it's a good idea! Though I'd love to suggest a slightly different order after the save game section:

- Screenshot
- Giant Screenshot
   `----------`
- Options
- Report a bug on Github
- About 'OpenRCT2'

This way it has slight order from casual and common usage to more about OpenRCT2 itself in the end. 🙂

**Comment by Gymnasiast** [comment]
I would prefer keeping Report a bug in the same place, and as a group of its own - makes it stand out more. Putting the screenshot options in a group of their own is fine by me, though.

**Reply by Basssiiie** [reply]
Wouldn't it be a bit overkill to have ~30-35% of the menu be separators, though? 😅

**Comment by AaronVanGeffen** [comment]
@Basssiiie:
> I think it's a good idea! Though I'd love to suggest a slightly different order after the save game section

Thanks. I agree with putting the screenshot options before the about/options section, yes.

@Gymnasiast:
> I would prefer keeping Report a bug in the same place, and as a group of its own - makes it stand out more.

I think the balance feels off to me precisely _because_ it stands out as an ungrouped item of its own. Personally, while I concede it warrants an item in the menu, I don't think it needs to stand out as much as it does.

> Putting the screenshot options in a group of their own is fine by me, though.

That's good to hear. I think it would not help balance much if we leave the 'report a bug' item as-is, though. If anything, I worry it might make it worse.
